app/assets/stylesheets/_plans.css.scss

```diff
@@ -30,3 +33,41 @@ h2 > .glyphicon-calendar {
 .toggle-section {
   appearance: none;
 }
+
+h2 > .glyphicon-calendar {
+  padding-left: 5px;
+  padding-top: 2px;
+}
+
+.quarter-courses {
+  display: flex;
+  flex-flow: row wrap;
+}
+
+.plan-taken-course {
+  background-color: $gray-lighter;
+}
+
+.plan-course {
+  border: 1px solid #ccc;
```

Comment:
Color literals like `#ccc` should only be used in variable declarations; they should be referred to via variable everywhere else.